src/pyodide/internal/asgi.py

```diff
@@ -25,7 +25,7 @@ def acquire_js_buffer(pybuffer):
 def request_to_scope(req, env, ws=False):
     from js import URL
 
-    headers = [tuple(x) for x in req.headers]
+    headers = [(k.lower().encode(), v.encode()) for k, v in req.headers]
```

Comment:
In lieu of adding a test, can we add a comment here with an example that cares about this being bytes so that we can add a test ourselves later?

Reply:
It assumes FastAPI so that would be a main concern, but its for ASGI spec though so should I add for FastAPI or just asserting for spec?
 
Like, anything for 
```
@app.get("/example")
  async def example(request: Request):
    request.headers.get("content-type") # this will error if header is not bytes.
```
perhaps?

Reply:
Also is there way to test this locally? I want to build miniflare with master workerd, but building workers-sdk doesnt works..

Reply:
Oh Thanks, now I can test few things.